src/Settings/Builders/Make.hs

```diff
@@ -13,5 +15,22 @@ makeBuilderArgs = do
     mconcat
         [ builder (Make gmpPath          ) ? pure ["MAKEFLAGS=-j" ++ t]
         , builder (Make libffiPath       ) ? pure ["MAKEFLAGS=-j" ++ t, "install"]
-        , builder (Make "testsuite/tests") ? pure ["THREADS=" ++ t, "fast"]
         ]
+
+validateBuilderArgs :: Args
+validateBuilderArgs = builder (Make "testsuite/tests") ? do
+    threads             <- shakeThreads <$> expr getShakeOptions
+    top                 <- expr topDirectory
+    compiler            <- expr $ fullpath ghc
+    checkPpr            <- expr $ fullpath checkPpr
+    checkApiAnnotations <- expr $ fullpath checkApiAnnotations
+    return [ "fast"
```

Comment:
Is the `"fast"` above related to `TestSpeed.Fast`?

Reply:
No, this is the default validate argument. Speed can be specified by running ./buils.dh test and not validate.

Reply:
Worst case, we can later arrange to support `--slow` and `--fast` flags, like the actual `validate` script, can't we?

Reply:
Yeah. I can do that right now.

Reply:
OK, I guess we can leave this for later.